packages/dd-trace/src/plugins/util/exec.js

```diff
@@ -1,10 +1,31 @@
 const cp = require('child_process')
 const log = require('../../log')
+const { distributionMetric, incrementCountMetric } = require('../../ci-visibility/telemetry')
```

Comment:
Why is a generic utility depending on CI visibility now?

Reply:
I see other similar issues across the board too.

Reply:
true. I'll remove `exec.js` and move `sanitizedExec` to `packages/dd-trace/src/plugins/util/git.js`, which is its only user.

Reply:
`agent-info-exporter.js` seems to also be using CI vis now.

Reply:
yeah but it only reports telemetry if in ci visibility mode: https://github.com/DataDog/dd-trace-js/pull/3752/files#diff-20cc60c898128ef54cc490f0a9ebaf3bfdd730571864431c9748425e39d37bd0R53